hack/qemu-metadata-api/server/server.go

```diff
@@ -110,6 +111,43 @@ func (s *Server) listPeers(w http.ResponseWriter, r *http.Request) {
 	log.Infof("Request successful")
 }
 
+// getEndpoint returns the IP address of the first control-plane instance.
+// This allows us to fake a load balancer for QEMU instances.
+func (s *Server) getEndpoint(w http.ResponseWriter, r *http.Request) {
```

Comment:
I like this route. We could even replace this part of the metadata API to return a real loadbalancer IP in the future without changing anything in the bootstrapper.